### PR TITLE
New scripts for checking services by nagios probes

### PR DESCRIPTION
**Pull request by stavamichal**
- add method getDestinationById to ServicesAgent
 - checkLastTaskResult script takes facilityId, serviceId and
   destinationId and checks if there is the newest taskResult
   not older then 2 days which ends without ERROR
 - listOfNRPECommands generates list of commands for NRPE server
   (nagios sond checking) in format "command[x]=pathToCheckScrips
   -options"

### CODE REVIEW

File: perun-cli/listOfNRPECommands [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    print help();
+    exit 0;
+  }) || die help();
+
+# MAIN CODE
+my @facilities = $facilitiesAgent->getFacilities;
+for my $facility (@facilities) {
+	my %facilityServices = ();
+	my $facilityId = $facility->getId;
+	my @resources = $facilitiesAgent->getAssignedResources( facility => $facilityId );
```
Comment (zlamalp) [inline perun-cli/listOfNRPECommands]: We have ``ServicesManager.getAssignedServices(Facility facility)`` in RPC API. If not present in CLI ServicesAgent, please add it and use it.

File: perun-cli/listOfNRPECommands [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+my $facilitiesAgent = $agent->getFacilitiesAgent;
+my $servicesAgent = $agent->getServicesAgent;
+my $resourcesAgent = $agent->getResourcesAgent;
+
+# CONSTANTS
+my $PERUN_CLI_PATH = '/opt/perun-cli/lib';
+my $CHECK_COMMAND = 'checkLastTaskResult';
+
+# HELP
+sub help {
```
Comment (zlamalp) [inline perun-cli/listOfNRPECommands]: Please fix alignment of ``--help`` output to match our standard tools.

File: perun-cli/checkLastTaskResult [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+my $agent = Perun::Agent->new();
+my $servicesAgent = $agent->getServicesAgent;
+my $propagationStatsReaderAgent = $agent->getPropagationStatsReaderAgent;
+
+# Constants
+my $CORRECT_STATUS = "DONE";
+my $MAX_TIME_FOR_TASK_RESULT = 172800;
+
+# HELP
+sub help {
```
Comment (zlamalp) [inline perun-cli/checkLastTaskResult]: Please fix alignment of ``--help`` output to match our standard tools.

**Comment by stavamichal** [comment]
Mentioned things are fixed now!

**Pull request merged**